tests/test_tiny_rwkv.c

```diff
@@ -28,7 +28,10 @@ void test_model(const char * model_path, const float * expected_logits, const fl
 
     struct rwkv_context * model = rwkv_init_from_file(model_path, N_THREADS);
     enum rwkv_error_flags error = rwkv_get_last_error(NULL);
-    ASSERT(error == 0, "Unexpected error %d", error);
+    ASSERT(model && error == 0, "Unexpected error %d", error);
+    struct rwkv_context * model2 = rwkv_clone_context(model, N_THREADS);
+    enum rwkv_error_flags error2 = rwkv_get_last_error(NULL);
+    ASSERT(model2 && error2 == 0, "Unexpected error2 %d", error2);
```

Comment:
I propose adding a separate test for cloning to avoid copy-paste here. It can be named like `test_context_cloning`, infer only `FP32` model and check that results are identical.

Reply:
done